src/framework/application.js

```diff
@@ -1136,14 +1136,15 @@ pc.extend(pc, function () {
                 if (assets[i].type === "texture" && assets[i].resource) {
                     assets[i].resource.destroy();
                 }
-                assets[i].unload();
+                assets[i].unloadOld();

```

Comment:
You used it somewhere in the app.destroy, calling resource.destroy manually, so I didn't change it.